Descent/DLittleLParser.class.st

```diff
@@ -102,6 +102,17 @@ DLittleLParser >> parse: aString [
 	^ self scanPair
 ]
 
+{ #category : #parsing }
+DLittleLParser >> parseMultiple: aString [ 
+	| result |
+	result := OrderedCollection new.
+	l := DImplicitStringLexer new.
+	l on: aString.
+	[[true] whileTrue: [ result add: self scanPair ]]

```

Comment:
This logic is dangerous because it relies on scanPair returning an error
I would prefer something like [ l atEnd ] whileFalse: [....]

Reply:
Resolved